modules/ml/src/main/java/org/apache/ignite/ml/composition/ModelsComposition.java

```diff
@@ -23,13 +23,15 @@
 import org.apache.ignite.ml.Exporter;
 import org.apache.ignite.ml.IgniteModel;
 import org.apache.ignite.ml.composition.predictionsaggregator.PredictionsAggregator;
+import org.apache.ignite.ml.environment.deploy.DeployableObject;
 import org.apache.ignite.ml.math.primitives.vector.Vector;
 import org.apache.ignite.ml.util.ModelTrace;
 
 /**
  * Model consisting of several models and prediction aggregation strategy.
  */
-public class ModelsComposition implements IgniteModel<Vector, Double>, Exportable<ModelsCompositionFormat> {
+public class ModelsComposition implements IgniteModel<Vector, Double>, Exportable<ModelsCompositionFormat>,
```

Comment:
Maybe we should make this class final too?

Reply:
I'm not sure, there is a child of this class (GDBModel) and I don't want to change an inheritance schema in this PR. Maybe we should create another ticker in Jira with a review of hierarchies of models (of course without changing APIs)